docs/04-Scripts.md

```diff
@@ -1,4 +1,159 @@
 # 4.0 - Scripts
 
+## 4.1 - Perl documentation
 
-![pipeline_flow](images/pipeline_schematic.png)
\ No newline at end of file
+All the perl scripts in the LORIS-MRI repository are documented. Users are 
+invited to consult the documentation 
+
+- at the terminal by typing:
+```angular2html
+perldoc %SCRIPT_NAME%
+```
+
+- in markdown format, on the [LORIS-MRI repository](../scripts_md/).
+
+Whether at the terminal or on-line, all scripts' documentations follow a used
+convention briefly outlined below:
+
+<pre>
+package XXXXX
+
+=pod
+
+=head1 NAME
+
+Name of the package or script
+
+=head1 SYNOPSIS
+
+A few lines of code on how to use the package or script
+
+=head1 DESCRIPTION
+
+Overall description of the package or script
+
+=head2 Methods
+
+=cut
+
+use warnings;
+use strict;
+
+=pod
+
+=head3 functionName($variable)
+
+Short description of the function.
+
+INPUT: short description of inputs or arguments
+
+RETURNS: short description of the returned objects
+
+=cut
+sub functionName{
+  ...
+}
+### REPEAT FUNCTION PART FOR EACH METHOD
+
+1; # end of script
+
+=pod
+
+=head1 TO DO
+
+Nothing planned (or things that are left to do)
+
+=head1 BUGS
+
+None reported (or list of bugs)
+
+=head1 LICENSING
+
+License: GPLv3
+
+=head1 AUTHORS
+
+LORIS community <loris.info@mcin.ca> and McGill Centre for Integrative 
+Neuroscience 
+
+=cut
+
+</pre>
+
+## 4.2 - Pipeline flow
+
+A very brief illustration of the key and expected outcomes at different steps in 
+the execution of the pipeline are shown in 
+![pipeline_flow](images/pipeline_schematic.png). This figure highlights the few 
```

Comment:
- Replace "shown in" by "shown below."
- Remove the "." after the link to the image 
- New line for "This figure highlights ... (otherwise "This" appears to the bottom right of the image in the MD display)

Reply:
done